lib/services/local_settings.dart

```diff
@@ -38,8 +38,6 @@ class LocalSettings {
 
   Future<void> _createDefault(key, value) async {
     var type = value.runtimeType.toString();
-    print("Key: $key , Val: $value, Type: $type");
-    print(type);
```

Comment:
Please restore the prints and change it to logger.d

Reply:
added back with logger.d

Reply:
https://github.com/STRDCoders/Ocnera/pull/31/commits/d073703cc952512224b92423b4f48d03117f0b6a